Tests/MapboxCoreNavigationTests/NativeHandlersFactoryTests.swift

```diff
@@ -61,7 +66,8 @@ class NativeHandlersFactoryTests: TestCase {
         ], forKey: customConfigKey)
         let expectedCustomConfig = [
             "features": [
-                "historyAutorecording": false
+                "historyAutorecording": false,
```

Comment:
Is the custom configuration format defined for public use? If so, we should at least define some constants for the keys, or better yet come up with a type-safe format for the configuration object.

Reply:
We have public but undocumented `customConfigKey` and `customConfigFeaturesKey` keys to work with custom `Navigator` config, but it is not intended for wide use. Currently it is more a developer or debug tool we left accessible for helping clients at some corner cases. IIRC there is no any kind of documentation about this config and it's capabilities.